maltoolbox/attackgraph/attackgraph.py

```diff
@@ -493,6 +333,176 @@ def attach_attackers(self) -> None:
 
             attacker.entry_points = list(attacker.reached_attack_steps)
 
+    def _follow_expr_chain(
+            self,
+            model: Model,
+            target_assets: list[Any],
+            expr_chain: ExpressionsChain
+        ) -> list:
+        """
+        Recursively follow a language graph expressions chain on an instance
+        model.
+
+        Arguments:
+        model           - a maltoolbox.model.Model on which to follow the
+                          expressions chain
+        target_assets   - the list of assets that this expressions chain
+                          should apply to. Initially it will contain the
+                          asset to which the attack step belongs
+        expr_chain      - the expressions chain we are following
+
+        Return:
+        A list of all of the target assets.
+        """
+
+        if expr_chain is None:
```

Comment:
Does this mean that the type annotation a few lines above should change to `Optional[ExpressionsChain]`?

Reply:
It should change and I will clarify the previous point as well with a comment in the code.